src/target/riscv/riscv.c

```diff
@@ -3675,6 +3700,16 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 			"expose. custom0 is accessed as abstract register number 0xc000, "
 			"etc. This must be executed before `init`."
 	},
+	{
+		.name = "hide_csrs",
+		.handler = riscv_set_hidden_csrs,
```

Comment:
For consistency, it would be good to call this `riscv_hide_csrs`.

```suggestion
		.handler = riscv_hide_csrs,
```

Reply:
addressed, thanks